pandas/core/indexes/datetimelike.py

```diff
@@ -82,21 +82,59 @@ class TimelikeOps(object):
 
     _round_doc = (
         """
-        %s the index to the specified freq
+        {op} the index to the specified `freq`.
 
         Parameters
         ----------
         freq : freq string/object
+            The frequency level to {op} the index to.
 
         Returns
         -------
-        index of same type
+        index of same type.
 
         Raises
         ------
-        ValueError if the freq cannot be converted
+        ValueError if the `freq` cannot be converted.
+
+        Notes
+        -----
+        See the following link for possible `freq` values
+        and their descriptions.\n
+        https://pandas.pydata.org/pandas-docs/stable/timeseries.html#offset-aliases
```

Comment:
You can use syntax like

    See :ref:`frequency aliases <timeseries.offset_aliases>` for a list of possible `freq` values.